<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="use_cpu"           value="true" />

  <!-- Start yolov8_ros iris_0 -->
  <node pkg="yolov8_ros" type="yolo_v8.py" name="yolov8_left_ros_iris_0" output="screen" >
    <param name="weight_path"     value ="$(find yolov8_ros)/weights/best.pt"/>
    <param name="image_topic"     value ="/iris_0/stereo_camera/left/image_raw" />
    <param name="pub_topic"       value ="/yolov8/iris_0/left/BoundingBoxes" />
    <param name="camera_frame"    value ="camera_color_frame"/>
    <param name="visualize"       value ="true"/>
    <param name="conf"            value ="0.3" />
  </node>

</launch>
